[2025-03-19T21:17:10.038+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: faud_detection_pipeline.load_data_hive scheduled__2023-10-23T12:00:00+00:00 [queued]>
[2025-03-19T21:17:10.100+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: faud_detection_pipeline.load_data_hive scheduled__2023-10-23T12:00:00+00:00 [queued]>
[2025-03-19T21:17:10.108+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-03-19T21:17:10.171+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): load_data_hive> on 2023-10-23 12:00:00+00:00
[2025-03-19T21:17:10.197+0000] {standard_task_runner.py:57} INFO - Started process 6013 to run task
[2025-03-19T21:17:10.221+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'faud_detection_pipeline', 'load_data_hive', 'scheduled__2023-10-23T12:00:00+00:00', '--job-id', '3226', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmp64hpsoza']
[2025-03-19T21:17:10.229+0000] {standard_task_runner.py:85} INFO - Job 3226: Subtask load_data_hive
[2025-03-19T21:17:10.420+0000] {task_command.py:415} INFO - Running <TaskInstance: faud_detection_pipeline.load_data_hive scheduled__2023-10-23T12:00:00+00:00 [running]> on host 445fc0f18383
[2025-03-19T21:17:11.002+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='faud_detection_pipeline' AIRFLOW_CTX_TASK_ID='load_data_hive' AIRFLOW_CTX_EXECUTION_DATE='2023-10-23T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-23T12:00:00+00:00'
[2025-03-19T21:17:11.005+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-19T21:17:11.007+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec hive hive -f /home/hive_script.hql']
[2025-03-19T21:17:11.074+0000] {subprocess.py:86} INFO - Output:
[2025-03-19T21:17:18.011+0000] {subprocess.py:93} INFO - SLF4J: Class path contains multiple SLF4J bindings.
[2025-03-19T21:17:18.012+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hive/lib/log4j-slf4j-impl-2.6.2.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-19T21:17:18.013+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hadoop-2.7.4/share/hadoop/common/lib/slf4j-log4j12-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-19T21:17:18.013+0000] {subprocess.py:93} INFO - SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[2025-03-19T21:17:18.015+0000] {subprocess.py:93} INFO - SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
[2025-03-19T21:17:23.179+0000] {subprocess.py:93} INFO - 
[2025-03-19T21:17:23.180+0000] {subprocess.py:93} INFO - Logging initialized using configuration in file:/opt/hive/conf/hive-log4j2.properties Async: true
[2025-03-19T21:18:12.509+0000] {subprocess.py:93} INFO - OK
[2025-03-19T21:18:12.516+0000] {subprocess.py:93} INFO - Time taken: 38.303 seconds
[2025-03-19T21:18:13.138+0000] {subprocess.py:93} INFO - OK
[2025-03-19T21:18:13.140+0000] {subprocess.py:93} INFO - Time taken: 0.618 seconds
[2025-03-19T21:18:13.213+0000] {subprocess.py:93} INFO - OK
[2025-03-19T21:18:13.215+0000] {subprocess.py:93} INFO - Time taken: 0.066 seconds
[2025-03-19T21:18:14.101+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-19T21:18:14.219+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=faud_detection_pipeline, task_id=load_data_hive, execution_date=20231023T120000, start_date=20250319T211710, end_date=20250319T211814
[2025-03-19T21:18:14.329+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-19T21:18:14.408+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
